website/docs/r/storage_object_replication_policy.html.markdown

```diff
@@ -0,0 +1,117 @@
+---
+subcategory: "Storage"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_object_replication_policy"
+description: |-
+  Manages a Storage Object Replication Policy.
+---
+
+# azurerm_storage_object_replication_policy
+
+Manages a Storage Object Replication Policy.
```

Comment:
could we elaborate on what this does? like "copes objets from a source storage account to a destination one ect"